main.py

```diff
@@ -32,11 +32,9 @@ class FrontmatterModel(BaseModel):
     """
 
     date: pydatetime.datetime | None
-    layout: str
     permalink: str | None
     redirect_from: list[str] | None
     redirect_to: str | None  # via the jekyll-redirect-from plugin
-    sitemap: bool | None
```

Comment:
The rest looked fine, but will data leak into our sitemap file without this? Previously, this was an issue, which is what led to it.

Reply:
that's a question for @mtrythall

Reply:
I remember @mtrythall saying he added support for sitemaps a month or two ago. If the new system has a better way, I just want to make sure we aren't leaking info to search engines before we are ready. (This was previously an issue we fixed)

Reply:
Added it back in just in case